package.json

```diff
@@ -46,7 +46,7 @@
     "knex": "^0.13",
     "mocha": "^2",
     "mysql": "^2",
-    "objection": "^0.8.0",
+    "objection": "github:Vincit/objection.js#d93d18e5220a83b9ef40ad3a53728403c77eed8c",
```

Comment:
Please note that this PR should not be merged before v0.9.0 is out, and then this line should be changed to "objection": "^0.9.0" before merging!

Reply:
Also, please note that I am working with yarn. I am not sure this statement works as it should with NPM.

Reply:
The syntax is the same with recent npm version.

Reply:
objection v0.9.0 has been released now.